modules/face/samples/sample_face_swapping.cpp

```diff
@@ -0,0 +1,195 @@
+#include "opencv2/face.hpp"
+#include "opencv2/imgcodecs.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/photo.hpp"
+
+using namespace cv;
+using namespace cv::face;
+using namespace std;
+unsigned long minv(unsigned long a,unsigned long b);
+void applyAffineTransform(Mat &warpImage, Mat &src, vector<Point2f> &srcTri, vector<Point2f> &dstTri);
+void calculateDelaunayTriangles(Rect rect, vector<Point2f> &points, vector< vector<int> > &delaunayTri);
+void warpTriangle(Mat &img1, Mat &img2, vector<Point2f> &t1, vector<Point2f> &t2);
+unsigned long minv(unsigned long a,unsigned long b){
```

Comment:
Why are we hardcoding the lookup of a min or maximum value? I would think that given an array of values the function minMaxLoc does everything your need?